src/test/resources/ingest-test-sample.json

```diff
@@ -1,7 +1,7 @@
 {"id":"sample1","participant_ids":["participant_1"],"date_collected":"2019-02-27","derived_from":null}
-{"id":"sample2","participant_ids":["participant_2"],"date_collected":"2019-02-27","derived_from":null}
-{"id":"sample3","participant_ids":["participant_3"],"date_collected":"2019-02-27","derived_from":null}
-{"id":"sample4","participant_ids":["participant_3"],"date_collected":"2019-02-27","derived_from":null}
-{"id":"sample5","participant_ids":["participant_2","participant_5"],"date_collected":"2019-02-27","derived_from":null}
-{"id":"sample6","participant_ids":[],"date_collected":"2019-02-27","derived_from":"sample5"}
-{"id":"sample7","participant_ids":[],"date_collected":"2019-02-27","derived_from":"sample2"}
+{"id":"sample2","participant_ids":["participant_2"],"date_collected":"2020-02-29","derived_from":null}
```

Comment:
Fun fact: BigQuery was smart enough to recognize that `2019-02-29` wasn't a real date, and rejected it as a bad row